go.mod

```diff
@@ -1,3 +1,9 @@
 module github.com/stmcginnis/gofish
 
 go 1.16
+
+require (
```

Comment:
These changes to `go.mod` and `go.sum` still need to be reverted.

Reply:
done